README.md

```diff
@@ -316,255 +316,42 @@ public class LibertyRestEndpoint {
 ```
 
 ## Creating Microservice-Two
-1. The instructions for creating microservice-two are the same as microservice-one navigate up one directory then into the `microservice-one/src/main/java/application/rest` directory and repeat steps 2-6 from the Creating Microservice-One.
+1. The instructions for creating microservice-two are the same as microservice-one navigate up one directory then into the `microservice-two/src/main/java/application/rest` directory and repeat steps 2-6 from the Creating Microservice-One.
 2. Now create a file called LiberyRestEndpoint.java and add the following code to create the REST API calls for your second microservice......NEW CODE TO BE ADDED HERE!!!
 
 ## Creating the Web Application
-Now that we've created a gateway and microservices A and B, it's time to build a webapp to be able access and easily display the information within these microservices. 
+Now that we've created a gateway and two microservices (A and B), a webapp is needed to visualise the information in these microservices. To make this easier, a webapp has already been fully developed for this example. 
+The webapp used in this example has been created using Angular CLI and is running on a Node server. 
 
-1. Firstly we need to create a basic AngularJS webapp
-2. Then we need to use this basic webapp skeleton and customise it to show the information we wish to access from the microservices
-3. Finally we need to call API requests, through our gateway service, to the two microservices to be able to access and then display this information on the fron-end to the client.
+In order to use the webapp provided, it is essential that the pre-requisites previously stated are installed first. 
 

```

Comment:
The Prereqs do talk about installing the angular CLI but it should not be a problem to mention it again if there is something specific you need to do for this app like as you say "enter into the folder"